shared/chat/conversation/list.desktop.js

```diff
@@ -142,6 +144,27 @@ class ConversationList extends Component<void, Props, State> {
     this._onScrollSettled()
   }, 100)
 
+  showPopup (message: Message, event: any) {
+    if (message.type !== 'Text') return
+    const clientRect = event.target.getBoundingClientRect()
```

Comment:
Do you mean getting the button element position in window coordinates instead of the mouse event target rect?